colorama/ansitowin32.py

```diff
@@ -66,13 +80,16 @@ def __init__(self, wrapped, convert=None, strip=None, autoreset=False):
         # to support the ANSI codes.
         conversion_supported = on_windows and winapi_test()
 
+        # Does this version of Win 10 support ANSI?
+        win_10_ansi_support = on_windows and check_win_10_ansi_support()
```

Comment:
Let's change the name to "windows_ansi_support", in case it will be more than just Windows 10 in the future :)